templates/procuration_manager/proposal.html.twig

```diff
@@ -0,0 +1,115 @@
+{% extends 'procuration_manager/_layout.html.twig' %}
+
+{% block procuration_manager_content %}
+    <div class="text--summary b__nudge--bottom-large">
+        <a href="{{ path('app_procuration_manager_proposals') }}" class="back-to-list icon--with-text">
+            {{ include('components/caret--left.html.twig') }}
+            Retour à la liste
+        </a>
+    </div>
+
+    <h3 class="b__nudge--bottom-large">
+        <span id="proxy-title">Mandataire n°{{ proxy.id }}</span>
+    </h3>
+
+    <hr />
+
+    <div class="l__row l__tablet--col l__row--v-stretch b__nudge--bottom-60 procuration-manager__proxy">
+        <div class="procuration-manager">
+            <h4 class="b__nudge--bottom">
+                Statut :
+                {% if not proxy.foundRequests.isEmpty %}
+                    {{ proxy.foundRequests.count > 1 ? '<br />' : '' }}
+                    {% for foundRequest in proxy.foundRequests %}
+                        Associé à la demande n°<a href="{{ path('app_procuration_manager_request', { id: foundRequest.id }) }}">{{ foundRequest.id }}</a><br />
+                    {% endfor %}
+                {% elseif proxy.disabled %}
+                    Désactivé (n'est plus associable)
+                {% else %}
+                    Disponible
+                {% endif %}
+            </h4>
+
+            <div class="profile-label">
+                Mandataire
+            </div>
+            <div class="profile-value" id="proxy-author">
+                {{ proxy.gender == 'male' ? 'Monsieur' : 'Madame' }} {{ proxy.firstNames }} {{ proxy.lastName }}
+            </div>
+
+            <div class="profile-label">
+                Coordonnées
+            </div>
+            <div class="profile-value">
+                Email :
+                <a href="mailto:{{ proxy.emailAddress }}" id="proxy-email">
+                    {{ proxy.emailAddress }}
+                </a>
+                {% if proxy.phone %}
+                <br />
+                Téléphone :
+                <a href="{{ proxy.phone|phone_number_format(constant('\\libphonenumber\\PhoneNumberFormat::RFC3966')) }}" id="proxy-phone">
```

Comment:
```suggestion
                <a href="tel:{{ proxy.phone|phone_number_format(constant('\\libphonenumber\\PhoneNumberFormat::RFC3966')) }}" id="proxy-phone">
```

Reply:
It's already added due to `PhoneNumberFormat::RFC3966`